core/services/versionchooser/frontend/index.html

```diff
@@ -233,8 +237,10 @@ <h3>Applying...</h3>
         loadAvailableversions() {
           // TODO: validate repository, treat error fetching
           this.loadingImages = true
+          this.error = null
           axios.get(`v1.0/version/available/${this.selectedImage}`).then((response) => {
```

Comment:
It seems this request would benefit from a timeout.

Reply:
I'll include it in the the vuetify rewrite.